types/testingutils/runner.go

```diff
@@ -18,13 +18,13 @@ var AttesterRunner7Operators = func(keySet *TestKeySet) ssv.Runner {
 }
 
 var ProposerRunner = func(keySet *TestKeySet) ssv.Runner {
-	return baseRunner(types.BNRoleProposer, ssv.ProposerValueCheckF(NewTestingKeyManager(), types.BeaconTestNetwork, TestingValidatorPubKey[:], TestingValidatorIndex, nil), keySet)
+	return baseRunner(types.BNRoleProposer, ssv.ProposerValueCheckF(NewTestingKeyManager(), types.BeaconTestNetwork, TestingValidatorPubKey[:], TestingValidatorIndex, nil, true), keySet)
 }
 
 var ProposerBlindedBlockRunner = func(keySet *TestKeySet) ssv.Runner {
 	ret := baseRunner(
 		types.BNRoleProposer,
-		ssv.ProposerValueCheckF(NewTestingKeyManager(), types.BeaconTestNetwork, TestingValidatorPubKey[:], TestingValidatorIndex, nil),
+		ssv.ProposerValueCheckF(NewTestingKeyManager(), types.BeaconTestNetwork, TestingValidatorPubKey[:], TestingValidatorIndex, nil, true),
```

Comment:
It is confusing that we have a blinded block value checker that works on the consensus and a blinded block runner that works on preconsesnsus...

Since those are 2 different things I guess it is fine...

Reply:
IIRC in pre-consensus phase they sign randao and once pre-consensus is reached, they get a block from node and run a consensus with the block data as consensus input, and that's where we need to handle a case if blinded support is off but a blinded block was received